packages/twig-to-php-parser/lib/twig-to-php-parser.php

```diff
@@ -282,7 +283,35 @@ function parse_svg_path( $twig_include, $svg_name, $is_using_plugin ) {
 		}
 	}
 
-	 return "<?php \PMC::render_template( " . $brand_directory . " . '" . $svg_directory . "' . ( $" . $svg_name . " ?? '' ) . '.svg', [], true ); ?>";
+	return "<?php \PMC::render_template( " . $brand_directory . " . '" . $svg_directory . "' . ( $" . $svg_name . " ?? '' ) . '.svg', [], true ); ?>";
+}
+
+/**
+ * Parser responsible for translation custom twig function: {{ wp_action( ... ) }}
+ * See larva/lib/server.js twing.addFunction for details
+ *
+ * @param string $twig_markup
+ * @return string
+ */
+function parse_wp_action( string $twig_markup )  : string {
```

Comment:
Can you add a Jest test for this? You can expose the `parse_wp_acton` method following the conventions in parser-method.js - this mixing of languages is a bit confusing / cumbersome, and something we will need to resolve eventually.